src/components/common/InsecureModeWarning.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <p>
+      Creating an address in the browser is unsafe. To offer you a secure
```

Comment:
Proposing a new warning (branding + extension is available)

```
Did you know that storing your key in the browser is considered unsafe? We offer a more secure way to storing your keys via our _browser extension_. Check it out in a quiet minute.
```

Reply:
I like that

Reply:
@mariopino did you see this comment?

Reply:
> @mariopino did you see this comment?

Yes but for some reason i thought it was already done.

What link should we include in ` _browser extension_` link? Chrome store?

Reply:
> > @mariopino did you see this comment?
> 
> Yes but for some reason i thought it was already done.
> 
> @faboweb What link should we include in ` _browser extension_` link? Chrome store?

Autoanswer: http://bit.ly/lunie-ext fits well here

Reply:
![image](https://user-images.githubusercontent.com/9256178/66825287-4c67cc00-ef4a-11e9-8b08-1d9a7cc38dfe.png)

Reply:
![image](https://media.giphy.com/media/xUOwG3MyaiItznFs6k/source.gif)